wix-scopes/src/index.js

```diff
@@ -0,0 +1,55 @@
+const SCOPES_TAG_NAME = 'scopes'
+const FRONTEND_SCOPE = 'frontend'
+const BACKEND_SCOPE = 'backend'
+const UNIVERSAL_SCOPES = [FRONTEND_SCOPE, BACKEND_SCOPE]
+
+function extractScopesFromTag(tag){
+  if(!tag || !tag.value){
+    return UNIVERSAL_SCOPES
+  }
+
+  const isScropesTagValid = tag.value.match(/^\[[a-zA-Z,]*\]$/)
+  if(!isScropesTagValid){
+    throw new Error(`scopes tag ${tag.value} is not valid`)
+  }
+
+  const scopesTagWithoutBrackets = tag.value.slice(1,tag.value.length-1)
+
+  if(!scopesTagWithoutBrackets){
+    throw new Error(`scopes tag ${tag.value} value is empty`)
+  }
+  const scopesValues = scopesTagWithoutBrackets.split(',')
+
+  for(const scopeValue of scopesValues){
+    if(!UNIVERSAL_SCOPES.includes(scopeValue)){
+      throw new Error(`unknown scope value ${scopeValue}`)
+    }
+  }
+
+  return scopesValues
+}
+
+exports.defineTags = function(dictionary) {
+  dictionary.defineTag(SCOPES_TAG_NAME, {
+    mustNotHaveValue : true,
```

Comment:
are u sure those values are correct? can u please check in the code what they aim to achieve?

Reply:
yep, they are correct, it is very clear in the docs, check it out if you want https://jsdoc.app/about-plugins.html

Reply:
@nemni8